### PR TITLE
[ticket/15661] Add core.viewtopic_modify_poll_ajax_data

### PR DESCRIPTION
**Pull request by senky**
PHPBB3-15661

Checklist:

- [x] Correct branch: master for new features; 3.2.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.2.x/development/git.html)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-15661

### CODE REVIEW

**Comment by senky** [comment]
!set Event

File: phpBB/viewtopic.php [hunk 1]
```diff
@@ -998,6 +998,29 @@
...
+			* Event to manipulate the poll data sent by AJAX response
+			*
+			* @event core.viewtopic_modify_poll_ajax_data
+			* @var	array	data				JSON response data
+			* @var	array	valid_user_votes	Valid user votes
+			* @var	array	vote_counts			Vote counts
+			* @var	array	forum_id			Forum ID
+			* @var	array	topic_data			Topic data
+			* @var	array	poll_info			Array with the poll information
+			* @since 3.2.3
```
Comment (Nicofuma) [inline phpBB/viewtopic.php]: 3.2.4 please

File: phpBB/viewtopic.php [hunk 2]
```diff
@@ -998,6 +998,29 @@
...
+			* @var	array	valid_user_votes	Valid user votes
+			* @var	array	vote_counts			Vote counts
+			* @var	array	forum_id			Forum ID
+			* @var	array	topic_data			Topic data
+			* @var	array	poll_info			Array with the poll information
+			* @since 3.2.4
+			*/
+			$vars = array(
+				'data',
+				'valid_user_votes',
```
Comment (rubencm) [inline phpBB/viewtopic.php]: this is included in $data

File: phpBB/viewtopic.php [hunk 3]
```diff
@@ -998,6 +998,29 @@
...
+			* Event to manipulate the poll data sent by AJAX response
+			*
+			* @event core.viewtopic_modify_poll_ajax_data
+			* @var	array	data				JSON response data
+			* @var	array	valid_user_votes	Valid user votes
+			* @var	array	vote_counts			Vote counts
+			* @var	array	forum_id			Forum ID
+			* @var	array	topic_data			Topic data
+			* @var	array	poll_info			Array with the poll information
+			* @since 3.2.4
```
Comment (marc1706) [inline phpBB/viewtopic.php]: 3.2.4-RC1

File: phpBB/viewtopic.php [hunk 4]
```diff
@@ -998,6 +998,29 @@
...
 			);
+
+			/**
+			* Event to manipulate the poll data sent by AJAX response
+			*
+			* @event core.viewtopic_modify_poll_ajax_data
+			* @var	array	data				JSON response data
+			* @var	array	valid_user_votes	Valid user votes
+			* @var	array	vote_counts			Vote counts
+			* @var	array	forum_id			Forum ID
```
Comment (marc1706) [inline phpBB/viewtopic.php]: Pretty sure it's not an array ;)

**Pull request merged**